arkane/referenceTest.py

```diff
@@ -137,5 +138,36 @@ def test_calculated_data_entry(self):
 
 
 
+class TestReferenceDatabase(unittest.TestCase):
+    """
+    Test that the ReferenceDatabase class functions properly
+    """
+
+    def test_load_main_reference_set(self):
+        """
+        Test that the main reference set can be loaded properly
+        """
+        database = ReferenceDatabase()
+        database.load()
```

Comment:
Consider separating into a setUpClass method if you think that additional methods will get added in the future.